### PR TITLE
Michal can't make it unfortunately, so replace with Max's talk

### PR DESCRIPTION
**Pull request by brian-brazil**
@juliusv

### CODE REVIEW

File: content/2017-munich/talks/improving-user-and-developer-experience-of-alertmanager-ui.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: Improving User and Developer Experience of Alertmanager UI
+---
+
+## Improving User and Developer Experience of Alertmanager UI
```
Comment (juliusv) [inline content/2017-munich/talks/improving-user-and-developer-experience-of-alertmanager-ui.md]: The title is grammatically weird: it seems to me it should either be "Alertmanager's UI" or "the Alertmanager UI". Probably the latter. /cc @mxinden

File: content/2017-munich/talks/improving-user-and-developer-experience-of-alertmanager-ui.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+Speaker: [Max Inden](/2017-munich/speakers/max-inden/)
+
+Alertmanager deduplicates, groups, and routs alerts from Prometheus to all
```
Comment (juliusv) [inline content/2017-munich/talks/improving-user-and-developer-experience-of-alertmanager-ui.md]: routs -> routes

**Comment by juliusv** [comment]
:+1: Otherwise.

File: content/2017-munich/talks/improving-user-and-developer-experience-of-alertmanager-ui.md (hunk 2 above)
Comment (brian-brazil) [inline content/2017-munich/talks/improving-user-and-developer-experience-of-alertmanager-ui.md]: Didn't you hear the new AM can make alerts beat a hasty retreat?

Comment (juliusv) [inline content/2017-munich/talks/improving-user-and-developer-experience-of-alertmanager-ui.md]: :laughing:

File: content/2017-munich/talks/improving-user-and-developer-experience-of-alertmanager-ui.md (hunk 1 above)
Comment (brian-brazil) [inline content/2017-munich/talks/improving-user-and-developer-experience-of-alertmanager-ui.md]: Given no feedback from Max, I'd say we merge this for now so the schedule is accurate.

**Pull request merged**